A hang on startup: NullPointerException - insuficient space in /tmp

**Describe the bug**
My client hangs on startup, before the GUI is even loaded. See logs below. Edit: Apparently this was caused by me having insufficient space /tmp partition.

**To Reproduce**
Reduce space on /tmp partition on a Linux system. Try to run runelite.

**Expected behavior**
Give a human-readable error message.

**Environment (please complete the following information):**
 - OS: Linux 5.1.11-arch1-1-ARCH
 - RuneLite version: 1:1.5.26-1

**Additional context**

stdout/stderr:

```
OpenJDK 64-Bit Server VM warning: Insufficient space for shared memory file:
   4409
Try using the -Djava.io.tmpdir= option to select an alternate temp location.

11:17:35,652 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback-test.xml]
11:17:35,652 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Could NOT find resource [logback.groovy]
11:17:35,652 |-INFO in ch.qos.logback.classic.LoggerContext[default] - Found resource [logback.xml] at [jar:file:/usr/share/runelite/RuneLite.jar!/logback.xml]
11:17:35,680 |-INFO in ch.qos.logback.core.joran.spi.ConfigurationWatchList@591f989e - URL [jar:file:/usr/share/runelite/RuneLite.jar!/logback.xml] is not of type file
11:17:35,784 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - debug attribute not set
11:17:35,801 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.core.ConsoleAppender]
11:17:35,807 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [STDOUT]
11:17:35,820 |-INFO in ch.qos.logback.core.joran.action.NestedComplexPropertyIA - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
11:17:35,924 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - About to instantiate appender of type [ch.qos.logback.core.rolling.RollingFileAppender]
11:17:35,934 |-INFO in ch.qos.logback.core.joran.action.AppenderAction - Naming appender as [FILE]
11:17:35,977 |-INFO in c.q.l.core.rolling.TimeBasedRollingPolicy@1711574013 - No compression will be used
11:17:35,980 |-INFO in c.q.l.core.rolling.TimeBasedRollingPolicy@1711574013 - Will use the pattern /home/jachym/.runelite/logs/client_%d{yyyy-MM-dd}.%i.log for the active file
11:17:35,985 |-INFO in ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP@61443d8f - The date pattern is 'yyyy-MM-dd' from file name pattern '/home/jachym/.runelite/logs/client_%d{yyyy-MM-dd}.%i.log'.
11:17:35,986 |-INFO in ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP@61443d8f - Roll-over at midnight.
11:17:35,993 |-INFO in ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP@61443d8f - Setting initial period to Thu Jun 20 11:17:29 CEST 2019
11:17:35,994 |-WARN in ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP@61443d8f - SizeAndTimeBasedFNATP is deprecated. Use SizeAndTimeBasedRollingPolicy instead
11:17:35,994 |-WARN in ch.qos.logback.core.rolling.SizeAndTimeBasedFNATP@61443d8f - For more information see http://logback.qos.ch/manual/appenders.html#SizeAndTimeBasedRollingPolicy
11:17:36,000 |-INFO in ch.qos.logback.core.joran.action.NestedComplexPropertyIA - Assuming default type [ch.qos.logback.classic.encoder.PatternLayoutEncoder] for [encoder] property
11:17:36,004 |-INFO in ch.qos.logback.core.rolling.RollingFileAppender[FILE] - Active log file name: /home/jachym/.runelite/logs/client.log
11:17:36,004 |-INFO in ch.qos.logback.core.rolling.RollingFileAppender[FILE] - File property is set to [/home/jachym/.runelite/logs/client.log]
11:17:36,006 |-INFO in ch.qos.logback.classic.joran.action.RootLoggerAction - Setting level of ROOT logger to INFO
11:17:36,006 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [STDOUT] to Logger[ROOT]
11:17:36,007 |-INFO in ch.qos.logback.core.joran.action.AppenderRefAction - Attaching appender named [FILE] to Logger[ROOT]
11:17:36,007 |-INFO in ch.qos.logback.classic.joran.action.ConfigurationAction - End of configuration.
11:17:36,008 |-INFO in ch.qos.logback.classic.joran.JoranConfigurator@445b84c0 - Registering current configuration as safe fallback point

2019-06-20 11:17:37 [main] WARN  n.r.client.discord.DiscordService - Failed to load Discord library, Discord support will be disabled.
2019-06-20 11:17:41 [main] INFO  net.runelite.client.rs.ClientLoader - client-patch 226435185.157
2019-06-20 11:17:42 [main] INFO  n.r.client.account.SessionManager - No session file exists
2019-06-20 11:17:43 [main] WARN  n.r.client.plugins.PluginManager - Error instantiating plugin!
net.runelite.client.plugins.PluginInstantiationException: com.google.inject.CreationException: Unable to create injector, see the following errors:

1) Error injecting constructor, java.lang.ExceptionInInitializerError
  at net.runelite.client.plugins.timetracking.clocks.ClockManager.<init>(Unknown Source)
  at net.runelite.client.plugins.timetracking.clocks.ClockManager.class(Unknown Source)
  while locating net.runelite.client.plugins.timetracking.clocks.ClockManager
    for field at net.runelite.client.plugins.timetracking.TimeTrackingPlugin.clockManager(Unknown Source)
  at net.runelite.client.plugins.PluginManager.lambda$instantiate$7(PluginManager.java:435)

1 error
	at net.runelite.client.plugins.PluginManager.instantiate(PluginManager.java:453)
	at net.runelite.client.plugins.PluginManager.scanAndInstantiate(PluginManager.java:288)
	at net.runelite.client.plugins.PluginManager.loadCorePlugins(PluginManager.java:197)
	at net.runelite.client.RuneLite.start(RuneLite.java:251)
	at net.runelite.client.RuneLite.main(RuneLite.java:221)
Caused by: com.google.inject.CreationException: Unable to create injector, see the following errors:

1) Error injecting constructor, java.lang.ExceptionInInitializerError
  at net.runelite.client.plugins.timetracking.clocks.ClockManager.<init>(Unknown Source)
  at net.runelite.client.plugins.timetracking.clocks.ClockManager.class(Unknown Source)
  while locating net.runelite.client.plugins.timetracking.clocks.ClockManager
    for field at net.runelite.client.plugins.timetracking.TimeTrackingPlugin.clockManager(Unknown Source)
  at net.runelite.client.plugins.PluginManager.lambda$instantiate$7(PluginManager.java:435)

1 error
	at com.google.inject.internal.Errors.throwCreationExceptionIfErrorsExist(Errors.java:470)
	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:176)
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:110)
	at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:232)
	at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:236)
	at net.runelite.client.plugins.PluginManager.instantiate(PluginManager.java:447)
	... 4 common frames omitted
Caused by: java.lang.ExceptionInInitializerError: null
	at net.runelite.client.plugins.timetracking.clocks.ClockManager.<init>(ClockManager.java:59)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.google.inject.internal.DefaultConstructionProxyFactory$ReflectiveProxy.newInstance(DefaultConstructionProxyFactory.java:126)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:111)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter$1.call(ProviderToInternalFactoryAdapter.java:46)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1092)
	at com.google.inject.internal.ProviderToInternalFactoryAdapter.get(ProviderToInternalFactoryAdapter.java:40)
	at com.google.inject.internal.SingletonScope$1.get(SingletonScope.java:194)
	at com.google.inject.internal.InternalFactoryToProviderAdapter.get(InternalFactoryToProviderAdapter.java:41)
	at com.google.inject.internal.SingleFieldInjector.inject(SingleFieldInjector.java:54)
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:132)
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:93)
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:80)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1085)
	at com.google.inject.internal.MembersInjectorImpl.injectAndNotify(MembersInjectorImpl.java:80)
	at com.google.inject.internal.Initializer$InjectableReference.get(Initializer.java:223)
	at com.google.inject.internal.Initializer.injectAll(Initializer.java:132)
	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:174)
	... 8 common frames omitted
Caused by: java.lang.NullPointerException: null
	at javax.swing.ImageIcon.<init>(ImageIcon.java:240)
	at net.runelite.client.plugins.timetracking.clocks.ClockTabPanel.<clinit>(ClockTabPanel.java:76)
	... 31 common frames omitted
2019-06-20 11:17:43 [main] INFO  n.r.c.plugins.kourendlibrary.Library - Library is now reset
2019-06-20 11:17:44 [main] WARN  n.r.client.plugins.PluginManager - Error instantiating plugin!
net.runelite.client.plugins.PluginInstantiationException: com.google.inject.CreationException: Unable to create injector, see the following errors:

1) Error injecting constructor, java.lang.NullPointerException
  at net.runelite.client.plugins.statusbars.StatusBarsOverlay.<init>(Unknown Source)
  while locating net.runelite.client.plugins.statusbars.StatusBarsOverlay
    for field at net.runelite.client.plugins.statusbars.StatusBarsPlugin.overlay(Unknown Source)
  at net.runelite.client.plugins.PluginManager.lambda$instantiate$7(PluginManager.java:435)

1 error
	at net.runelite.client.plugins.PluginManager.instantiate(PluginManager.java:453)
	at net.runelite.client.plugins.PluginManager.scanAndInstantiate(PluginManager.java:288)
	at net.runelite.client.plugins.PluginManager.loadCorePlugins(PluginManager.java:197)
	at net.runelite.client.RuneLite.start(RuneLite.java:251)
	at net.runelite.client.RuneLite.main(RuneLite.java:221)
Caused by: com.google.inject.CreationException: Unable to create injector, see the following errors:

1) Error injecting constructor, java.lang.NullPointerException
  at net.runelite.client.plugins.statusbars.StatusBarsOverlay.<init>(Unknown Source)
  while locating net.runelite.client.plugins.statusbars.StatusBarsOverlay
    for field at net.runelite.client.plugins.statusbars.StatusBarsPlugin.overlay(Unknown Source)
  at net.runelite.client.plugins.PluginManager.lambda$instantiate$7(PluginManager.java:435)

1 error
	at com.google.inject.internal.Errors.throwCreationExceptionIfErrorsExist(Errors.java:470)
	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:176)
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:110)
	at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:232)
	at com.google.inject.internal.InjectorImpl.createChildInjector(InjectorImpl.java:236)
	at net.runelite.client.plugins.PluginManager.instantiate(PluginManager.java:447)
	... 4 common frames omitted
Caused by: java.lang.NullPointerException: null
	at net.runelite.client.util.ImageUtil.resizeImage(ImageUtil.java:191)
	at net.runelite.client.plugins.statusbars.StatusBarsOverlay.<init>(StatusBarsOverlay.java:98)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.google.inject.internal.DefaultConstructionProxyFactory$ReflectiveProxy.newInstance(DefaultConstructionProxyFactory.java:126)
	at com.google.inject.internal.ConstructorInjector.provision(ConstructorInjector.java:111)
	at com.google.inject.internal.ConstructorInjector.construct(ConstructorInjector.java:90)
	at com.google.inject.internal.ConstructorBindingImpl$Factory.get(ConstructorBindingImpl.java:268)
	at com.google.inject.internal.SingleFieldInjector.inject(SingleFieldInjector.java:54)
	at com.google.inject.internal.MembersInjectorImpl.injectMembers(MembersInjectorImpl.java:132)
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:93)
	at com.google.inject.internal.MembersInjectorImpl$1.call(MembersInjectorImpl.java:80)
	at com.google.inject.internal.InjectorImpl.callInContext(InjectorImpl.java:1085)
	at com.google.inject.internal.MembersInjectorImpl.injectAndNotify(MembersInjectorImpl.java:80)
	at com.google.inject.internal.Initializer$InjectableReference.get(Initializer.java:223)
	at com.google.inject.internal.Initializer.injectAll(Initializer.java:132)
	at com.google.inject.internal.InternalInjectorCreator.injectDynamically(InternalInjectorCreator.java:174)
	... 8 common frames omitted
2019-06-20 11:17:44 [main] ERROR net.runelite.client.RuneLite - Uncaught exception:
java.lang.ExceptionInInitializerError: null
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at net.runelite.client.plugins.PluginManager.instantiate(PluginManager.java:424)
	at net.runelite.client.plugins.PluginManager.scanAndInstantiate(PluginManager.java:288)
	at net.runelite.client.plugins.PluginManager.loadCorePlugins(PluginManager.java:197)
	at net.runelite.client.RuneLite.start(RuneLite.java:251)
	at net.runelite.client.RuneLite.main(RuneLite.java:221)
Caused by: java.lang.NullPointerException: null
	at net.runelite.client.util.ImageUtil.resizeCanvas(ImageUtil.java:206)
	at net.runelite.client.plugins.poison.PoisonPlugin.<clinit>(PoisonPlugin.java:73)
	... 10 common frames omitted
```

--- a/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
+++ b/runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java
@@ -52,6 +52,11 @@ import net.runelite.api.SpritePixels;
 @Slf4j
 public class ImageUtil
 {
+	static
+	{
+		ImageIO.setUseCache(false);
+	}
+
 	/**
 	 * Creates a {@link BufferedImage} from an {@link Image}.
 	 *

